cmd/server.go

```diff
@@ -105,7 +113,9 @@ var Serve = &cobra.Command{
 		forward(e, "/config", configDb)
 		forward(e, "/canary", api.CanaryCheckerPath)
 		forward(e, "/kratos", kratosAPI)
-		forward(e, "/apm", api.ApmHubPath)
+		forward(e, "/apm", api.ApmHubPath) // Deprecated
+
+		forwardWithMiddlewares(e, "/logs", api.ApmHubPath, logSearchMiddleware)
```

Comment:
I don't think this a middleware - those are normally generic -

Reply:
modified to use a http handler instead